test/EndaomentAdmin.test.js

```diff
@@ -0,0 +1,265 @@
+const { accounts, contract } = require("@openzeppelin/test-environment");
+const {
+  BN,
+  constants,
+  expectEvent,
+  expectRevert,
+} = require("@openzeppelin/test-helpers");
+const { assert } = require("chai");
+
+const EndaomentAdmin = contract.fromArtifact("EndaomentAdmin");
+const OrgFactory = contract.fromArtifact("OrgFactory");
+const FundFactory = contract.fromArtifact("FundFactory");
+
+describe("EndaomentAmin.sol", function() {
+  const [admin, pauser, accountant, reviewer, newOwner] = accounts;
+
+  beforeEach(async function() {
+    this.endaomentAdmin = await EndaomentAdmin.new({ from: admin });
+  });
+
+  it("has defined contract address post-init", async function() {
+    assert.isDefined(this.endaomentAdmin.address);
+  });
+
+  it("set originator as the Owner on construct, allows retrieval of owner address", async function() {
```

Comment:
simplify name

Reply:
which name?

Reply:
set originator as the Owner on construct, allows retrieval of owner address

Reply:
like you think this could be worded better basically? yea?